packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -792,6 +794,39 @@ export class ConversationAPI {
     return response.data;
   }
 
+  /**
+   * Post an encrypted message to a conversation.
+   * @param messageData Mls message payload in TLS format. Please refer to the MLS specification for details.
+   * @see https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-message-framing
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/postOtrMessage
+   */
+  public async postMlsMessage(messageData: Uint8Array): Promise<MlsEvent> {
```

Comment:
is there a possibility the data type could also be a string? it was the case for Otr messages, not sure how its working for Mls @atomrc

Reply:
For OTR messages, that was no string, it was a Protobuf that contained a `string`. 
But it gets encoded into a Uint8Array right before sending :) (see https://github.com/wireapp/wire-web-packages/blob/main/packages/api-client/src/conversation/ConversationAPI.ts#L673)